code/game/gamemodes/cult/ritual.dm

```diff
@@ -210,20 +209,19 @@
 			if(!(CULT_ELDERGOD in cult_mode.objectives) || !(CULT_SLAUGHTER in cult_mode.objectives))
 				to_chat(user, "<span class='warning'>[cult_mode.cultdat.entity_name]'s power does not wish to be unleashed!</span>")
 				return 0
-			var/confirm_final = alert(user, "This is the FINAL step to summon your dietys power, it is a long, painful ritual and the crew will be alerted to your presence", "Are you prepared for the final battle?", "My life for Nar-Sie!", "No")
-			if(confirm_final == "No")
-				to_chat(user, "<span class='cult'>You decide to prepare further before scribing the rune.</span>")
-				return
-			else
-				return 1
+		var/confirm_final = alert(user, "This is the FINAL step to summon your dietys power, it is a long, painful ritual and the crew will be alerted to your presence", "Are you prepared for the final battle?", "My life for [cult_mode.cultdat.entity_name]!", "No")
+		if(confirm_final == "No")

```

Comment:
This doesn't use a "Yes" button, though, it uses a "My life for (god name)!" button.

Could just assign the string to a var and use that, I guess.